### PR TITLE
Update Website Styles!!!

### LINKED ISSUE #28: Update styles to new design!
Update the layout/content/styles to fit the [new design by Jamie](https://www.figma.com/file/0dnG0Wk1kBGnprclGlpQrl/Social-Justice-Hackathon-Design-Edits?type=design&node-id=13-10952&mode=design&t=iaKtymCzcl9VtqCQ-0)

image
image

### PR DESCRIPTION
**Pull request by travissouthard**
# Overview

Updates the styles and content to fit Jamie's design (linked in #28)

Closes #28 

# Demo

Before
image

After
image

# Testing

- Either locally or with the "view deployment" feature, look through the pages and ensure:
  - [ ] The content is all correct
  - [ ] The site is readable
  - [ ] The designs fit [Jamie's designs](https://www.figma.com/file/0dnG0Wk1kBGnprclGlpQrl/Social-Justice-Hackathon-Design-Edits?type=design&node-id=13-10952&mode=design&t=ej812faEhm58lUoj-0) (There are a couple things we can't adjust as granularly since Quarto is a little odd)

### CODE REVIEW

**Comment by insiyahj** [comment]
Requests for PR 29: 
1. Please see edits on the screenshots attached 
2. Updated url for header https://www.eventbrite.com/e/philly-open-data-data-analysis-and-engineering-best-practices-and-pizza-tickets-860772532817
3. Updated text for under the large graphic:
Students, programmers, “civic-minded hackers”, faculty, justice advocates, public servants, community legal providers, writers, story-tellers, and members of the Philly community will all come together to design creative solutions to respond to specific justice needs of residents and neighborhoods most impacted by social, economic, and environmental injustices in Philadelphia. This event is free to attend, and open to all participants, regardless of coding experience or affiliation with an organization or university. See you soon!

image

image

File: styles.scss [hunk 1]
```diff
@@ -115,6 +159,31 @@ a {
...
 
+.sponsor-logos {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-evenly;
+  img {
+    max-height: 50px;
+  }
+}
+
```
Comment (machow) [inline styles.scss:185]: It looks like on narrow screens, the footer link text does some aggressive wrapping.

image

I was able to---for narrow displays---get the links to each be on their own line, using this rough css:

```suggestion

@media (max-width: 720px) {
    .nav-footer-left {
      flex: none !important;

      .footer-items {
        display: block !important;
      }
    }

}

```

File: styles.scss [hunk 2]
```diff
@@ -68,10 +92,24 @@ a {
...
 .hero-banner {
-  background-image: url('./assets/about-page_hero-banner.svg');
-  background-size: auto 100%;
+  background-image: url('./assets/bg.png');
+  background-size: cover;
   background-position: center;
+  color: white;
+  height: 480px;
+
```
Comment (machow) [inline styles.scss:121]: On wide displays, the headers aren't each on a newline. "Philly's 2nd" and Social Justice Hackathon are put side-by-side.

image

The suggested CSS below ensures it wraps on wide displays

```suggestion
  display: flex;
  /* Uncomment below to vertically center text */
  /*justify-content: center;*/
  align-items: center;
  flex-direction: column;
  /* outline letters in black */
  /* from https://stackoverflow.com/a/4772914 */
  text-shadow: 1px 0px 0 #000, 0 -1px 0 #000, 0 1px 0 #000, -1px 0 0 #000;

```

Here is the result:

image

Comment (machow) [reply styles.scss:121]: Note that text-shadow is a little bit funky for anything other than 1 px outline. There's the `-website-text-stroke` property that's pretty widely supported, but I haven't tried that.

File: index.qmd [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    HTML("April 5th-7th @ Drexel Kline School of Law")
+)
+```
+:::
+
+<!-- /HERO BANNER -->
+
+:::::: {.content-block}
+:::{.hero-text .content-block}
+  The Philadelphia Social Justice Hackathon will be held at the Drexel Law School from April 5-7, 2024, to bring students, programmers, and other “civic-minded hackers” together – to work in collaboration with justice advocates, public servants, community legal providers, and members of the community.
```
Comment (machow) [inline index.qmd]: From [this slack thread](https://codeforphilly.slack.com/archives/C06DXBH1HE2/p1711299903315159?thread_ts=1711248924.227479&cid=C06DXBH1HE2)

```suggestion
Students, programmers, “civic-minded hackers”, faculty, justice advocates, public servants, community legal providers, writers, story-tellers, and members of the Philly community will all come together to design creative solutions to respond to specific justice needs of residents and neighborhoods most impacted by social, economic, and environmental injustices in Philadelphia.
```

**Review by machow — CHANGES_REQUESTED** [review]
This is looking great! I added suggestions for CSS to ensure...

* banner text wraps on wide displays
* footer links aren't side-by-side on narrow displays (which made their link text wrap).

Feel free to punt on / adapt the suggestions!

edit: Insiyah flagged the text visibility in the hero banner, which seems important. I've added a black outline to the text in my CSS suggestions. I'd suggest we merge with less than ideal visibility, and seek someone to take that specific issue head on. (It's worth their full attention / a limited scope PR)

**Comment by travissouthard** [comment]
Merging for times' sake but all notes here and in the [website slack channel](https://codeforphilly.slack.com/archives/C06DXBH1HE2). Any further changes should be noted in new issues.

**Pull request merged**